test/Driver/driver-time-compilation.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %swiftc_driver -parse -driver-time-compilation %s 2>&1 | %FileCheck %s
+
+// CHECK: Driver Time Compilation
+// CHECK: Total Execution Time: {{[0-9]+}}.{{[0-9]+}} seconds ({{[0-9]+}}.{{[0-9]+}} wall clock)
+// CHECK: ---Wall Time---
+// CHECK: --- Name ---
+// CHECK: driver-time-compilation.swift
+// CHECK: {{[0-9]+}}.{{[0-9]+}} (100.0%)  Total
```

Comment:
Looks good! I think it's worth having a multi-file test too. You could just add an extra RUN line that uses {{%S/../Inputs/empty.swift}}, a standard empty .swift file, as well as {{%s}}. (Both tests are useful.)